Tests/BackendTest/BlockChangeRequestTest.cs

```diff
@@ -0,0 +1,128 @@
+using FluentAssertions;
+using FRITeam.Swapify.Backend;
+using FRITeam.Swapify.Entities;
+using FRITeam.Swapify.Entities.Enums;
+using MongoDB.Driver;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace BackendTest
+{
+    [Collection("Database collection")]
+    public class BlockChangeRequestText : IClassFixture<Mongo2GoFixture>
+    {
+        private readonly Mongo2GoFixture _mongoFixture;
+
+        public BlockChangeRequestText(Mongo2GoFixture mongoFixture)
+        {
+            _mongoFixture = mongoFixture;
+        }
+
+        [Fact]
+        public async Task ExchangeRequest()
+        {
+            IMongoDatabase database = _mongoFixture.MongoClient.GetDatabase("StudentsDB");
```

Comment:
the test body is too long. Create multiple small tests not one big. Add also Negative Tests.

Reply:
test is long because I need a lot of object before I will start testing exchange